proof-systems/Cargo.toml

```
@@ -33,6 +33,7 @@ poly-commit = { path = "./src/darlin/poly-commit", optional = true }
 r1cs-std = { path = "../r1cs/gadgets/std", optional = true }
 
 rand = { version = "=0.8.4" }
+rand_core = { version = "0.6.3" }
```

```suggestion
rand_core = { version = "=0.6.3" }
```

From now onwards, let's use fixed dependencies when adding new ones or updating already existing ones